src/style.css

```diff
@@ -51,7 +51,7 @@ body {
   display: flex;
   align-items: center;
   border: 1px solid #f6f6f6;
-  cursor: move;
+  /* cursor: move; */
 }
 
 .entry,
```

Comment:
- [ ] when I click on the checkbox, I should see a strikethrough on the word, please fix this as well